From c2ab12b3742c929a225c3753439438edc27bfa81 Mon Sep 17 00:00:00 2001
From: Stanislav Ochotnicky <sochotnicky@redhat.com>
Date: Tue, 1 Feb 2011 14:42:27 +0100
Subject: [PATCH] Fix qt4 pinentry window created in the background

This is probably just a workaround. Proper fix is being investigated.
See:
https://bugzilla.redhat.com/show_bug.cgi?id=589532
http://stackoverflow.com/questions/2788518/calling-activatewindow-on-qdialog-sends-window-to-background
---
 qt4/pinentrydialog.cpp |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/qt4/pinentrydialog.cpp b/qt4/pinentrydialog.cpp
index 541baf4..d634eb6 100644
--- a/qt4/pinentrydialog.cpp
+++ b/qt4/pinentrydialog.cpp
@@ -69,7 +69,6 @@ void raiseWindow( QWidget* w )
     SetForegroundWindow( w->winId() );
 #endif
     w->raise();
-    w->activateWindow();
 }
 
 QPixmap icon( QStyle::StandardPixmap which )
-- 
1.7.3.5

